metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -105,6 +107,21 @@ object UserConfiguration {
         |`src/main/scala:: src/main/java::`. Syntax such as `src/{main,test}::`
         |is not supported.
         |""".stripMargin
+    ),
+    UserConfigurationOption(
+      "bloop-sbt-already-installed",
+      "false",
+      "false",
+      "Don't generate Bloop plugin file for Sbt",
+      "Option that can specified to true if the user doesn't want to generate an sbt file adding the Bloop sbt plugin."
```

Comment:
```suggestion
      "If true, Metals will not generate a `project/metals.sbt` file under the assumption that sbt-bloop is already manually installed in the sbt build. Build import will fail with a 'not valid command bloopInstall' error in case Bloop is not manually installed in the build when using this option."
```